packages/rocketchat-slackbridge/slackbridge.js

```diff
@@ -13,6 +13,12 @@ class SlackBridge {
 		this.rocket.setSlack(this.slack);
 		this.slack.setRocket(this.rocket);
 
+		//Settings that we cache versus looking up at runtime
```

Comment:
Just added these here at the top for quick access to properties being used in SlackBridge